frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -77,6 +77,57 @@ function Application({ getWidgetQueryBuilder }: DashboardProps): JSX.Element {
 		[servicename, topLevelOperations, tagFilterItems, getWidgetQueryBuilder],
 	);
 
+	const onTracePopupClick = (timestamp: number): void => {
```

Comment:
@KolesnikAntony is this https://github.com/SigNoz/signoz/blob/develop/frontend/src/container/MetricsApplication/Tabs/util.ts#L38 the same util ?

Reply:
Probably made mistake when was merging it. Will fix